src/ert/storage/local_experiment.py

```diff
@@ -37,6 +37,7 @@
     GenKwConfig.__name__: GenKwConfig,
     SurfaceConfig.__name__: SurfaceConfig,
     FieldConfig.__name__: FieldConfig,
+    "Field": FieldConfig,
```

Comment:
Also need to ensure that "Field" is written to storage instead of "FieldConfig".

Reply:
I don't think that's necessary?
If it writes FieldConfig, FieldConfig lookup will give ...`FieldConfig`.
If it has written (with old storage version), Field, it will also give `FieldConfig`

Reply:
I think over time, trying to support older versions in this way is going to get hairy. I would rather we continued to migrate rather than keep aliases.